src/java.desktop/share/native/libfontmanager/freetypeScaler.c

```diff
@@ -137,6 +143,8 @@ static void invalidateJavaScaler(JNIEnv *env,
                                  FTScalerInfo* scalerInfo) {
     freeNativeResources(env, scalerInfo);
     (*env)->CallVoidMethod(env, scaler, invalidateScalerMID);
+    // NB: Exceptions must not be cleared (and therefore no JNI calls performed) after calling this method
```

Comment:
Please split long lines to 80 chars per line

Reply:
Done.